src/Runner/Interpreters/ZendPhpCliInterpreter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of the Nette Tester.
+ * Copyright (c) 2009 David Grudl (https://davidgrudl.com)
+ */
+
+namespace Tester\Runner\Interpreters;
+
+
+/**
+ * Zend PHP CLI executable.
+ */
+class ZendPhpCliInterpreter extends AbstractInterpreter
+{
+
+	public function __construct($path, $args = NULL)
+	{
+		parent::__construct($path, ' -n' . $args);
```

Comment:
If args are specified, space will be missing after `-n` (applies to all variants above/below).